### DISCUSSION TITLE
SSH Login

### DISCUSSION THREAD

**Discussion by wallace4793** [1000001101000/Debian_on_Buffalo#174] [General]
Hi,

I managed to get the debain installed using your buster install files (rather than the other ones), it worked great. However at the ssh login neither the root or user name/pass I used during install works. I get access denied with worng credentials. I redid the whole thing again and wiped the hdd and started fdrom scratch making the user and password simple so I was sure i wasnt getting it wrong but its the same the second time around... is there another login I should use?

John

**Comment by 1000001101000** [comment]
What model device is this?

Usually this is a sign that the /boot wasn't configured in the partitioning screen. If this is the case the installer/install credentials will likely still work since it would be booting back into the installer.

**Comment by wallace4793** [comment]
Its a Linkstation WXL. You are correct, I can ssh in to the installer. Given this has failed twice, can I do something from here to correct or do i have to start over?

Thanks for your help. John

**Comment by 1000001101000** [comment]
You should be able to move forward with the install normally. This time when you get to the partition menu you will need to configure /boot as described here: https://github.com/1000001101000/Debian_on_Buffalo/wiki/Running-the-installer
